### PR TITLE
Exclude lock balance for staking asset while saving user balance in store

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Store will save user balance... for staking asset, it will check if lock balance exist... If yes, then it will exclude that amount and then store the balance in the store..
This will prevent user from adding liquidity from staked/locked balance

### CODE REVIEW

File: src/redux/epics/exchange/updatePayAssetBalance.epic.ts [hunk 1]
```diff
@@ -31,10 +32,23 @@ export const updateAssetsBalanceEpic = (
                 ],
                 store,
             ]): Observable<Action<any>> => {
-                return api.query.genericAsset.freeBalance(assetId, signingAccount).pipe(
-                    switchMap((balance: Balance) => {
-                        const userBal = new Amount(balance);
-                        const newAssetBalance = {assetId: assetId, account: signingAccount, balance: userBal};
+                return combineLatest([
+                    api.query.genericAsset.freeBalance(assetId, signingAccount),
+                    api.query.genericAsset.stakingAssetId(),
```
Comment (holygits) [inline src/redux/epics/exchange/updatePayAssetBalance.epic.ts:37]: can 'staking asset id' go in the global state store

**Review by holygits — APPROVED** [review]
lgtm

File: src/redux/epics/exchange/updatePayAssetBalance.epic.ts (hunk 1 above)
Comment (KarishmaBothara) [reply src/redux/epics/exchange/updatePayAssetBalance.epic.ts:37]: ya... I can add it to global...use here from global store..

**Comment by KarishmaBothara** [comment]
Local storage key tested for three chains
0.36.1-Development-EXTENSION_META_UPDATED (Local chain)
0.36.1-CENNZnet Nikau-EXTENSION_META_UPDATED (Nikau)
0.36.1-CENNZnet Azalea-EXTENSION_META_UPDATED (Azalea)

**Review by eupston — APPROVED** [review]
lgtm

**Comment by KarishmaBothara** [comment]
Update local storage key to

1. 0.36.1-Development40-EXTENSION_META_UPDATED (local chain)
2. 0.36.1-CENNZnet Azalea40-EXTENSION_META_UPDATED (Main chain)
3. 0.36.1-CENNZnet Nikau40-EXTENSION_META_UPDATED (Test chain)

**Pull request merged**